src/Pages/Login.js

```diff
@@ -0,0 +1,100 @@
+import "./authentication.css";
+import { useState } from "react";
+import { Navbar } from "../Components/Navbar/Navbar";
+import { useAuth } from "../context/auth-context";
+import { loginService } from "../services/loginService";
+import { Link } from "react-router-dom";
+const Login = () => {
+  const [loginInput, setLoginInput] = useState({
+    input: {},
+    error: "",
+    hide: { pwd: true },
+  });
+  const { setIsAuth, setToken, navigate } = useAuth();
+
+  const loginInputHandler = (e) => {
+    const { name, value } = e.target;
+    setLoginInput({
+      ...loginInput,
+      input: { ...loginInput.input, [name]: value },
+    });
+  };
+
+  const loginHandler = async (e) => {
+    e.preventDefault();
+    try {
+      const { encodedToken } = await loginService(loginInput.input);
+      localStorage.setItem("isAuth", true);
+      localStorage.setItem("token", encodedToken);
+      setToken(encodedToken);
+
+      setLoginInput({ ...loginInput, input: { email: "", password: "" } });
+      setIsAuth(true);
+
+      navigate(-1);
+    } catch (err) {
+      setLoginInput({
+        ...loginInput,
+        error: "You must have entered wrong credentials!!",
```

Comment:
You are setting a credential message for each error that is wrong. You should have set a message for a specific error.